libraries/AP_RangeFinder/AP_RangeFinder_Lua.h

```diff
@@ -20,15 +20,22 @@ class AP_RangeFinder_Lua : public AP_RangeFinder_Backend
     void update(void) override;
 
     // Get update from Lua script
-    bool handle_script_msg(float dist_m) override;
+    bool handle_script_msg(const RangeFinder::RangeFinder_State &state_arg) override;
```

Comment:
Why not a smaller, writable-only struct? But perhaps this is the Lua binding design pattern, I don't know.